<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexMainWindow</class>
 <widget class="QMainWindow" name="IndexMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IndexMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_base_area">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_submit"/>
   <addaction name="action_indexing"/>
   <addaction name="action_check_local"/>
   <addaction name="action_check_local_with_update"/>
   <addaction name="action_total_number"/>
   <addaction name="action_auto_indexing"/>
   <addaction name="action_undo_auto_indexing"/>
   <addaction name="action_show_original_data"/>
   <addaction name="action_show_bottom_view"/>
   <addaction name="action_show_error_view"/>
   <addaction name="action_show_help_view"/>
  </widget>
  <widget class="QToolBar" name="toolBar_advanced_area">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_set_font"/>
   <addaction name="action_text_replace"/>
   <addaction name="action_advanced_setting"/>
   <addaction name="action_save_data"/>
   <addaction name="action_get_new_task"/>
  </widget>
  <widget class="QToolBar" name="toolBar_search_area">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_search"/>
   <addaction name="action_reset_search_result"/>
  </widget>
  <widget class="QToolBar" name="toolBar_label_area">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_combobox_area">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="action_submit">
   <property name="text">
    <string>提交</string>
   </property>
   <property name="toolTip">
    <string>提交到缓存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_check_local">
   <property name="text">
    <string>待提交不刷新</string>
   </property>
   <property name="toolTip">
    <string>切换视图到缓存数据</string>
   </property>
   <property name="shortcut">
    <string>Alt+Z</string>
   </property>
  </action>
  <action name="action_check_local_with_update">
   <property name="text">
    <string>待提交刷新</string>
   </property>
   <property name="toolTip">
    <string>切换视图到缓存数据，同时更新缓存数据</string>
   </property>
  </action>
  <action name="action_total_number">
   <property name="text">
    <string>统计总数</string>
   </property>
  </action>
  <action name="action_auto_indexing">
   <property name="text">
    <string>批量智能标记</string>
   </property>
   <property name="toolTip">
    <string>通过一定算法实现智能标引</string>
   </property>
  </action>
  <action name="action_undo_auto_indexing">
   <property name="text">
    <string>撤销批量标记</string>
   </property>
  </action>
  <action name="action_show_original_data">
   <property name="text">
    <string>改变状态</string>
   </property>
   <property name="toolTip">
    <string>显示原始引文数据</string>
   </property>
  </action>
  <action name="action_show_bottom_view">
   <property name="text">
    <string>显示原始引文</string>
   </property>
   <property name="toolTip">
    <string>显示隐藏下方视图</string>
   </property>
  </action>
  <action name="action_show_error_view">
   <property name="text">
    <string>显示错误</string>
   </property>
   <property name="toolTip">
    <string>显示错误视图</string>
   </property>
  </action>
  <action name="action_show_help_view">
   <property name="text">
    <string>标记提示</string>
   </property>
   <property name="toolTip">
    <string>显示操作帮助提示</string>
   </property>
  </action>
  <action name="action_set_font">
   <property name="text">
    <string>字体</string>
   </property>
   <property name="toolTip">
    <string>设置字体</string>
   </property>
  </action>
  <action name="action_text_replace">
   <property name="text">
    <string>文本替换</string>
   </property>
   <property name="toolTip">
    <string>按照设定好的文本进行自动替换</string>
   </property>
  </action>
  <action name="action_advanced_setting">
   <property name="text">
    <string>高级</string>
   </property>
   <property name="toolTip">
    <string>高级设置</string>
   </property>
  </action>
  <action name="action_save_data">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前数据到数据库</string>
   </property>
  </action>
  <action name="action_get_new_task">
   <property name="text">
    <string>取任务</string>
   </property>
   <property name="toolTip">
    <string>获取新任务数据</string>
   </property>
  </action>
  <action name="action_search">
   <property name="text">
    <string>查找</string>
   </property>
   <property name="toolTip">
    <string>查找指定文本</string>
   </property>
  </action>
  <action name="action_reset_search_result">
   <property name="text">
    <string>还原</string>
   </property>
   <property name="toolTip">
    <string>重置查找之后文本显示样式</string>
   </property>
  </action>
  <action name="action_indexing">
   <property name="text">
    <string>正加标记</string>
   </property>
   <property name="toolTip">
    <string>显示引文标记视图</string>
   </property>
   <property name="shortcut">
    <string>Alt+Z</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
